torch/_inductor/bisect_helper.py

```diff
@@ -468,7 +468,7 @@ def __del__(self) -> None:
                     cls.bisection_enabled = bisection_enabled_orig
                     cls.delete_bisect_status()
 
-            cleanup = DisableBisect()
```

Comment:
@albanD @Skylion007 It was this line.  Removing the local variable `cleanup` but keeping the calculation worked on my development environment but failed every time on CI!